data/frontier jobs.txt

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2017 by Michael Zahniser and Nate Graham
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+mission "Instruct novice pilot"
+	job
+	repeat
+	description "A novice pilot seeks instruction from an experienced spacer, and requests an escort to <stopovers>. After that, return to <origin> and you will receive payment of <payment>."
+	to offer
+		random < 5
+		"combat rating" > 50
+	source
+		government Republic "Free Worlds" Neutral
+		attributes frontier "dirt belt" rim north south mining farming
+	stopover
+		attributes frontier
+		distance 3 7
+	on stopover
+		dialog "After a smooth landing, you guide your student through an especially rough part of the spaceport, full of trash, homeless drifters, and poorly-lit alleyways. You explain the ins and outs of avoiding pickpockets, keeping out of bar fights, and how to spot a lousy deal at an outfitter's shop. After a few hours in one of the seedier parts of the galaxy, both of you are eager to return to <origin>."
+	on visit
+		dialog "You've returned to <planet>, but forgot to escort the <npc> to <stopovers> first! Better head there first."
+	on complete
+		payment 75000
+		dialog "Your student thanks you for the able tutelage and for providing a guided view of some of the galaxy's rougher areas. You receive your payment of <payment>."
+	npc
+		government Pirate
+		personality nemesis waiting
+		fleet "Small Northern Pirates"
+		system
+			distance 3 7
+	npc accompany save
+		government Merchant
+		personality escort heroic vindictive
+		fleet
+			names "civilian"
+			variant
+				"Sparrow"
+			variant
+				"Star Barge"
+			variant
+				"Shuttle"
+			variant
+				"Heavy Shuttle"
+
+
+
+mission "Small Pirate gambling"
+	description "Survive the pirate attack!"
+	minor
+	repeat
+	to offer
+		"reputation: Pirates" < 0
+		"combat rating" < 10
+		random < 5
```

Comment:
there is a `credits` condition now, so you can ensure this offers properly: #2497